tests/cupyx_tests/scipy_tests/fftpack_tests/test_fftpack.py

```diff
@@ -125,6 +147,47 @@ def test_ifft_overwrite_plan(self, xp, scp, dtype):
                                  overwrite_x=True)
         return x
 
+    @testing.for_complex_dtypes()
+    @testing.numpy_cupy_allclose(rtol=1e-4, atol=1e-7, accept_error=ValueError,
+                                 contiguous_check=False, scipy_name='scp')
+    def test_ifft_plan_manager(self, xp, scp, dtype):
+        x = testing.shaped_random(self.shape, xp, dtype)
+        # hack: avoid testing the cases when the output array is of size 0
+        # because cuFFT and numpy raise different kinds of exceptions
+        if self.n == 0:
+            return x
+        x_orig = x.copy()
+        if scp is cupyx.scipy:
+            from cupy.cuda.cufft import get_current_plan
+            plan = scp.fftpack.get_fft_plan(x, shape=self.n, axes=self.axis)
+            with plan:
+                assert id(plan) == id(get_current_plan())
+                out = scp.fftpack.ifft(x, n=self.n, axis=self.axis)
+            assert get_current_plan() is None
+        else:  # scipy
+            out = scp.fftpack.ifft(x, n=self.n, axis=self.axis)
+        testing.assert_array_equal(x, x_orig)
+        return out
+
+    @testing.for_complex_dtypes()
+    def test_fft_multiple_plan_error(self, dtype):
+        # hack: avoid testing the cases when the output array is of size 0
+        # because cuFFT and numpy raise different kinds of exceptions
+        if self.n == 0:
+            return
+        import cupy
+        import cupyx.scipy.fftpack as fftpack
+        x = testing.shaped_random(self.shape, cupy, dtype)
+        plan = fftpack.get_fft_plan(x, shape=self.n, axes=self.axis)
+        try:
+            with plan:
+                fftpack.fft(x, n=self.n, axis=self.axis, plan=plan)
+        except RuntimeError as e:
+            if not e.__str__().startswith('Use the cuFFT plan'):
```

Comment:
How about using `self.assertRaises`?

Reply:
I guess you meant `with pytest.raises(...):`? According to
https://docs-cupy.chainer.org/en/stable/contribution.html#how-to-write-tests:
> Use `with pytest.raises(...):` instead of `with self.assertRaises(...):`.

Reply:
Using `with pytest.raises(...):` is indeed better than `try...except...`, as the code is shorter and cleaner. Changes are made.

Reply:
`with pytest.raises(...)` is better. Thanks!